spec/views/courses/available.html.erb_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+RSpec.describe "home/available", type: :view do
+  let(:valid_session) { {} }
+
+
+
+  context "for students" do
+    before(:each) do
+      @course = FactoryBot.create(:course)
+      login_student
+    end
+
```

Comment:
Please add a test that verifies, that only courses are displayed, that the student is not enrolled in.